translations/validators.fr.yml

```diff
@@ -78,6 +78,7 @@ committee.must_accept_contacting_terms: Vous devez accepter d'être contacté(e)
 
 committee.event.invalid_category: Le type d'événement sélectionné n'est pas valide.
 committee.event.invalid_date_range: La date de fin de l'événement doit être postérieure à la date de début.
+committee.event.invalid_finish_date: La date de fin de votre événement ne peut pas dépasser le {{ limit }}.
```

Comment:
I guess we could have the same message ID for committees and CP. Not a big deal.

Reply:
the message changes `votre événement` and `votre action citoyenne`